src/options/options.css

```diff
@@ -128,22 +133,13 @@
 }
 
 header {
-    position: absolute;
-    left:0;
-    right: 0;
-    top:0;
```

Comment:
need to check

Reply:
Ok. I got it.
Actually, the original intention was to have a fixed layout with a scroll bar in the list area, but apparently, I didn't do it 😛.
I will fix it later. Thank you for cleanup.